src/main/java/uk/gov/hmcts/darts/common/service/impl/DataAnonymisationServiceImpl.java

```diff
@@ -68,22 +95,22 @@ public void anonymizeHearingEntity(UserAccountEntity userAccount, HearingEntity
     @Override
     public void anonymizeEventEntity(UserAccountEntity userAccount, EventEntity eventEntity) {
```

Comment:
Although not related to this PR be good to make these private or default scoped. Could lead to bugs if the caller assumes that the db persisting saving will be done for them. This is not the case

Reply:
I'm happy to default scope these for now can always pull them back up downstream if they are needed.
Updated